setup-cli/README.md

```diff
@@ -0,0 +1,28 @@
+# setup-cli
+
+This `opslevel/actions/setup-cli` actions is a [composite action](https://docs.github.com/en/actions/creating-actions/about-custom-actions#composite-actions)
+that sets up the OpsLevel CLI in your GitHub Actions workflow.
+
+## Usage
+
+The default configuration installs the latest version of OpsLevel CLI:
+
+```yaml
+steps:
+- uses: opslevel/actions/setup-cli@main
+```
+
+A specific version of OpsLevel CLI can be installed:
+
+```yaml
+steps:
+- uses: opslevel/actions/setup-cli@main
```

Comment:
should this be `@main` or should we start cutting releases?

Reply:
We should probably start cutting releases. I can add that as part of this PR